### PR TITLE
Normalize require as import

### PR DESCRIPTION
**Pull request by MarcoWorms**

### CODE REVIEW

File: src/react-event-components/index.js [hunk 1]
```diff
@@ -1,14 +1,14 @@
-const KeyUp = require('./KeyUp')
-const KeyDown = require('./KeyDown')
-const Every = require('./Every')
-const DeviceOrientation = require('./DeviceOrientation')
-const TouchStart = require('./TouchStart')
-const TouchMove = require('./TouchMove')
-const TouchEnd = require('./TouchEnd')
-const TouchCancel = require('./TouchCancel')
-const GeolocationChange = require('./GeolocationChange')
+import KeyUp from './KeyUp'
```
Comment (grvcoelho) [inline src/react-event-components/index.js]: We could use `export` syntax here.

Example: `export Keyup from './KeyUp`

File: src/react-event-components/DeviceOrientation.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-const { Component, PropTypes } = require('react')
+import { Component, PropTypes } from 'react'
```
Comment (felquis) [inline src/react-event-components/DeviceOrientation.js]: I love it!

**Comment by felquis** [comment]
Error while testing this PR

```shell
git checkout master
git fetch origin
git checkout normalize-imports
yarn
```

image

While with `npm install`

image

I heard @derekstavis mentioning something about it yesterday

**Comment by derekstavis** [comment]
Yeah @felquis, it happens because `npm` and `yarn` runs `prepublish` task after `install`. This is probably done by design on this two package managers, but makes absolutely no sense for me.

The task is failing because I have appended a `git diff --quiet` on `prepublish` task, to avoid building and pushing dirty state to npm. If you try again `npm install` or `yarn` on a clean repository (I see you have a X mark on your prompt) the task will not error.

We can further discuss and opt to remove it if it's boring us :stuck_out_tongue:

**Comment by felquis** [comment]
git status

```shell
	modified:   dist/react-event-components.js
	modified:   dist/react-event-components.js.map
	modified:   dist/react-event-components.min.js
	modified:   dist/react-event-components.min.js.map
```

**Comment by derekstavis** [comment]
Oh, sure. Then we must add `dist/` to `.gitignore` and use `git add --force` when commiting an updated distribution. Also to note is that we should never commit `dist/` on default workflow. `dist/` should be force added and commited only on version bump/tag commits.

File: src/react-event-components/index.js (hunk 1 above)
Comment (derekstavis) [inline src/react-event-components/index.js]: Probably slipped out of @MarcoWorms sight :stuck_out_tongue:

**Comment by felquis** [comment]
@derekstavis rebased this PR, CI is pending.

**Comment by derekstavis** [comment]
10/10 :ship:

**Pull request merged**